src/go/rpk/pkg/cli/transform/meta.go

```diff
@@ -81,3 +81,40 @@ Subsequent 'rpk transform list' operations will show transform processors as
 	}
 	return cmd
 }
+
+func newSetCompressionCommand(fs afero.Fs, p *config.Params) *cobra.Command {
```

Comment:
Probably confusing for users that this is not sticky across deploys

Reply:
I am not sure! I just wanted to point out that the following sequence of commands would end up with a non expected outcome.

```
rpk transform deploy
rpk transform set compression=gzip
rpk transform deploy
# no more gzip!
```

I personally would just not have this command, but otherwise I think writing back to the file is a reasonable choice (or adding a warning if there is no config file)

Reply:
No, it's a great point. The `set` command doesn't seem particularly useful. Thanks!